New package: PowerModelsGMD v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: PowerModelsGMD
- Repository: https://github.com/lanl-ansi/PowerModelsGMD.jl
- Created by: @bluejuniper
- Version: v1.0.0
- Commit: 500e2fd1f71ca0d3a43714fe32254a9958c61333
- Reviewed by: @bluejuniper
- Reference: https://github.com/lanl-ansi/PowerModelsGMD.jl/issues/82#issuecomment-2420367780
- Description: GMD problem formulation for PowerModels.jl
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598c874a8cfe5f41b81dd2005f4e817181aa042044122f931015e6d12c61b1f1cfc7be1f0b5587654d4a9a27996c410a4cbde59c7a786bd1d4ae03c4b7d4a15d94e24cfd0b6a1331adcf309c6ecf9147dc385d497d27f9e36fcc21002959db3bcb11e3c6a6006018c1ca483374fc28c1d697a229077626c491fd69485c9afb47de0 -->

briochemc commented:
[noblock]

Is "GMD" a well known acronym? If not I would recommend spelling it out as "GeomagneticDisturbances" as in "PowerModelsGeomagneticDisturbances". GMD is also an acronym for the [Geoscientific Model Development journal](https://www.geoscientific-model-development.net/) and there may be other things called GMD.

goerz commented:
I think the name is fine. This package builds on PowerModels.jl, and that should be sufficient context. Pretty much PowerModelsAnything should receive very little scrutiny as far as the package name is concerned. [noblock]

briochemc commented:
[noblock] Oh I was unaware of PowerModels.jl. Feel free to ignore my earlier comment :)

JuliaRegistrator committed:
New package: PowerModelsGMD v1.0.0

UUID: c92a0612-66bb-11e9-0491-518d0d864356
Repo: https://github.com/lanl-ansi/PowerModelsGMD.jl.git
Tree: 710a9ef138915b6273cd1cd33876a1119daaeacd

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
